Intro to Data Science videos.Rmd

```diff
@@ -0,0 +1,8 @@
+# Intro to Data Science -- videos
+  
+Yucen Wang (yw3561)  
+
+The videos contain two parts. They explain the main idea of data science, data analytics and some considerations around big data.  
+
+[Intro to DS part 1](https://drive.google.com/file/d/1Qj3bdFzc7wfitP_NlPA7DAq0rrhAXccU/view?usp=sharing)  
```

Comment:
Please place additional resources (videos) in the resouces folder and link it through your .rmd file.

- All of the additional resources should be in the resources/<project_name>/ folder.
- Links to internal files must contain resources/<project_name>/ in the path, such as: ![Test Photo](resources/sample_project/election.jpg)